packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -863,6 +867,18 @@ class FlutterError extends Error with DiagnosticableTreeMixin implements Asserti
   /// recommended.
   static FlutterExceptionHandler onError = (FlutterErrorDetails details) => presentError(details);
 
+  /// Called by the Flutter framework before attempting to parse a [StackTrace].
+  ///
+  /// Some [StackTrace] implementations have a different toString format from
+  /// what the framework expects, like ones from package:stack_trace. To make
+  /// sure we can still parse and filter mangled [StackTrace]s, the framework
+  /// first calls this function to demangle them.
+  ///
+  /// This should be set in any environment that could propagate a non-standard
+  /// stack trace to the framework. Otherwise, the default behavior is to assume
+  /// all stack traces are in a standard format.
+  static StackTraceDemangler demangleStackTrace = (StackTrace e) => e;
```

Comment:
nit: Same doc nit, this might be a better place for an example than the typedef

nit: avoid abbreviations, `(StackTrace stackTrace) => stackTrace;`